nuxeo-drive-client/nxdrive/data/ui5/activity.html

```diff
@@ -21,7 +21,7 @@
 </script>
 </head>
 <body style="overflow: hidden;" ng-app="DriveActivity" ng-controller="DriveActivityCtl">
-	<div drive-include="getTemplate('activity-content')">
+	<div drive-include="getTemplate('activity-content.html')">
```

Comment:
The problem isnt more the fact that the file in template does not contain the .html extension ?